packages/react-components/react-tags/stories/Tag/TagDefault.stories.tsx

```diff
@@ -1,147 +1,157 @@
 import * as React from 'react';
-import { Tag, TagProps } from '@fluentui/react-tags';
+import { Tag, TagContent, TagProps } from '@fluentui/react-tags';
 import { Calendar3Day20Regular } from '@fluentui/react-icons';
 import { Avatar } from '@fluentui/react-components';
 
 // TODO I added many examples here for easier implementation. This story will be simplified to keep only the default example
 export const Default = (props: Partial<TagProps>) => (
   <div style={{ display: 'flex', columnGap: 28 }}>
     <div style={{ display: 'flex', flexDirection: 'column', rowGap: 28 }}>
-      <Tag
-        media={
-          <Avatar
-            {...{
-              name: 'Katri Athokas',
-              badge: { status: 'busy' },
-              image: {
-                src: 'https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/KatriAthokas.jpg',
-              },
-            }}
-          />
-        }
-        secondaryText="Secondary text"
-        dismissable={true}
-        {...props}
-      >
-        Primary text
+      <Tag dismissible {...props}>
+        <TagContent
```

Comment:
I would recommend to convert this PR to draft or don't merge this until you're sure you can go the route of

```tsx
<Tag>
  <TagContent />
  <TagDismiss />
</Tag>
```

Without that that API shape this is essentially adding an extra JSX element

Reply:
Converted, will split out the tagContent part to a separate effort and re-publish

Reply:
Updated, now there's only change for `interactive` in this PR